### ISSUE TITLE
Turrican Anthology Vol. 2 - 010061D0130CA000 - game freezes at the beginning

### ISSUE THREAD

**Issue by legolas119** [Ryujinx/Ryujinx-Games-List#4939]
### Game Name

Turrican Anthology Vol. 2

### Game Update Version

1.0.0

### Emulator Build Version

1.1.1291

### Graphics Backend

OpenGL and Vulkan

### Hardware Specs

- ##### CPU: *(AMD Ryzen 3700X)*
- ##### GPU: *(RTX 2060 Super)*
- ##### GPU Driver version: *(552.22)*
- ##### RAM: *(16GB)*

### Game State

The game freezes as soon as i run the game. If i try with Vulkan, there is a black screen, if i use Open GL it freezes at FACTOR 5 logo

### Screenshots

https://i.postimg.cc/s2hPTQkF/Turrican-Vol-2.png

**Comment by Eli0t77** [comment]
weird i completed the game on an older version, do you have a log file?

just tested it again on 1.1.1291 and its still working

-  CPU: i7-13700KF
-  GPU: NVIDIA RTX 4070 Ti
-  GPU Driver version: 551.52
-  RAM: 32GB

**Comment by legolas119** [comment]
yes sure i added the log now. It was not possible to drag& drop it when i opened the ticket. Do you notice any issue in my log file? I have same problem with TUrrican Collection Vol 1
thank you

**Comment by Eli0t77** [comment]
the only thing i noticed was that yours loads as nsp and mine is an xci cart dump
i didn't get Vol 1 as i already had Turrican Flashback

**Comment by legolas119** [comment]
i tried the XCI version too, i attached the log, but i have same issue..
[Ryujinx_1.1.1291_2024-04-25_19-50-28.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15118838/Ryujinx_1.1.1291_2024-04-25_19-50-28.log)
